src/client/css/fonts.css

```diff
@@ -105,6 +105,14 @@ https://fonts.googleapis.com/css2?family=Inter:wght@400;600&display=swap
   unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
 }
 
+@font-face {
+  font-family: Inter;
+  font-style: normal;
+  font-weight: 500;
+  font-display: optional;
+  src: url('../fonts/Inter-Medium.woff2') format('woff2');
+}
+
```

Comment:
Oh, are there UX review meetings? If so, could you invite me to those? I haven't really interacted with UX yet, so it would be great to be able to ask questions about things like this.

I removed both this font and the ones for the breaches empty state in af1df92b. Note that this does mean that the user's email address is no longer called out (i.e. bolded), and it's looking quite a bit heavier:

![image](https://user-images.githubusercontent.com/4251/218449701-459de174-0b42-4fde-8ad3-ed7d807bafdd.png)

Reply:
Yeah, I think we would at least need one other weight for Metropolis in order to use it typographically as intended or fallback use Inter in this instance. I agree that we should be cautions of resources being loaded if we only need them in single instances. Cutting back too far on design decisions is not necessarily the best solution in my opinion. Let’s consolidate with UX and make sure we understand implications and come to a decision together.

Reply:
Just a follow up on this particular case -- I reviewed it with UX and it was updated via https://github.com/mozilla/blurts-server/pull/2828

The solution agreed with UX was to use the paragraph text styles we already had in place (16px non-bold inter).  Apparently this should have already been in the designs, but content created this section instead of design, and so there were discrepencies. 

Currently the repo is set to use Metropolis for bolded headings only, and inter for paragraph with all styles allowed (bold/italic).  If we notice design going against this pattern, let's flag and discuss!

Reply:
Ah OK, that's good to know, thanks for sharing!